build/helper/metadata_merge_dicts.py

```diff
@@ -22,11 +22,14 @@ def merge_dicts(into, outof):
                 for item2 in outof[item]:
                     into[item][item2] = outof[item][item2]
             else:
-                # Handle regex in addon
-                for item2 in into:
-                    if re.search(item, item2):
-                        assert type(into[item2]) is dict
-                        merge_dicts(into[item2], outof[item])
+                # attributes keys are integers so they do not need the regex check (and
```

Comment:
One thing I noticed about this piece of code, even though it is not directly related to this change, is that substrings match.

So if you have:
"ConfigureVoltage" in the add ons, then it will match "ConfigureVoltageLevel" which may not be desirable.
Ideally you'd be forced to write "ConfigureVoltage.*" if you want to match both.

I saw this in NI-FGEN with "Initiate" matching "InitiateGeneration". I don't think it was intentional. Something to think about.